smarts/core/road_map.py

```diff
@@ -46,15 +46,19 @@ class RoadMap:
 
     @property
     def source(self) -> str:
+        """The road map resource source. Generally a file URI."""
         raise NotImplementedError()
 
     @property
-    def bounding_box(self) -> BoundingBox:
-        # may return None to indicate the map is unbounded.
+    def bounding_box(self) -> Optional[BoundingBox]:
+        """The minimum bounding box that contains the map geometry. May return `None` to indicate
+        the map is unbounded.
+        """
         raise NotImplementedError()
 
     @property
     def scale_factor(self) -> float:
+        """The ratio between the default lane width and the actual lane width"""
```

Comment:
Suggestion:  The ration between 1 unit on the map and 1 meter.

Reply:
```suggestion
        """The ratio between 1 unit on the map and 1 meter."""
```